### DISCUSSION TITLE
Perverted questions

### DISCUSSION THREAD

**Discussion by hlepnamingblyat** [Monika-After-Story/MonikaModDev#7908] [issue discussions]
Will it be able to ask Monika, some perv questions?
Since she asks if I pleasure myself, and to keep it a secret, so long as she is the one I'm thinking about, maybe we could ask her these types of questions too!
Like her sizes (especially bra), how many kids does she want to have with the player, does she pleasure herself (backfire).... ❤️ 
This may sound rude yeah... but it's human! Where would we be without some good ol' NSFW content? 😝 
Anyways, I really hope this comes to the game. I would really love to see her face when I ask this! 😆

**Comment by Freshairkaboom** [comment]
From my understanding, it's already implied that Monika does not know what pleasuring yourself feels like from the question "is it really that good of a feeling?". The kids question is definitely a topic I would want brought up at some point, especially with how Monika may or may not want them, and maybe talk about adoption or something. That could be a cool separate topic.

As for bra size, I'm not that interested myself, but if someone made a topic regarding it and made it good or interesting, and tied it into a conversation about size not mattering or something (lol), then I could see it work.

I think it's only natural to be interested in your girlfriend in that way too, but I definitely think these topics need to be reserved for higher affections, as to not have them be the front and center reason you play the mod.

Remember, Monika is a rather innocent 18 year old girl that's never been in a relationship before. As long as we remain tactful about it, exploring these sides of Monika shouldn't be a problem in my eyes.

EDIT: It was brought to my attention that a previous PR about kids was rejected because of the controversial outcome of such a topic if handled poorly. I'm more interested in a topic where the player gets to tell Monika about their own kids if they have them, but that too needs to be handled correctly, as to not bring the topic into whether or not Monika wants them.

**Comment by lunulae** [comment]
Since there's an 18+ filter for outfits I could see some more risque topics being handled in a similar way, though we'll probably want to be careful with those. The mod direction definitely isn't to go too explicit, even if it has an age check.

I'm wondering if it might be a good idea to have additional flags for if these topics would be available. It seems like the sort of thing that might be better to have as opt-in content rather than on by default. I know for example the paraphilia topic has a "I'd rather not talk about things like this" flag, which could be utilized?

**Comment by multimokia** [comment]
Definitely think a flag like that is needed. I know for a fact that:
a) Yes, people may be underage, so these things shouldn't be brought up
b) People who *are* of age may not feel comfortable talking about it.

If this does go into official as a thing, I feel like it'd definitely need a topic where Monika asks if you're okay with talking about this (A simple setting in the settings menu would be a bit immersion breaking and imo makes it feel less like communication between partners. Aside: Perhaps sensitive mode could do this too?)

If not official, then as a potential submod for this would make sense as it's purely opt in.

I'm a little torn personally on which way to go with it. 

On one hand, official makes sense in terms of relationship progression, and with the gateways handled properly, I feel like it can be viable.

On the other hand, it's a potential *slippery slope* depending on whatall possible topics may wind up contributed and we'd need to think out some new lines to draw for things in regards to Monika's character and general mod focus.

**Comment by Freshairkaboom** [comment]
Yeah, it's definitely a bit of a slippery slope, considering we don't have official source material to guide us during those topics, other than building upon her personality from existing topics. It's normal for a person to evolve, but I think the point of MAS was always to create a feeling that you're talking to the same Monika from the original game. This is why I don't like to write submod topics for my own use very much, because at that point there's no filter to prevent me from simply putting my own spin on Monika and changing who she is.

The way to go is probably to put these topics behind a barrier topic, like the quiz I mentioned in #6164, or something similar. There should probably also be a time check in addition to an affection check, like after your second anniversary or something. Also a way to prevent these topics from being brought up for a while if time travel has been involved, as I know these topics would probably be a way for...certain Youtubers to gain clickbait titles while hurting Monika in the process.

**Comment by EllieWLMS** [comment]
1 she loves us no matter what...why do we have to care about her bra size?
2 That's a good idea! 
3 I thought that she said in the same conversation that she didn't know what it feels like
4 Maybe she could talk about sexual education like Masturbation,ways to avoid infections, protection or even Female period
I'm sorry im Greek and i dont talk english...

**Comment by Freshairkaboom** [comment]
1. I don't know, so you can gift her underwear, sports bras or swimsuits for holidays or birthdays? Lots of reasons why you should get your girlfriend's sizes. Of course it's a perverted question too, which is kind of the point I guess. Something for a little further into the relationship, you know. With your permission, so she doesn't bring it up randomly, or you don't get to ask her right away or if you're uncomfortable seeing such questions.

**Comment by EllieWLMS** [comment]
No, it doesn't make me uncomfortable, Monika knows that she is in a game, We don't need to know her size.... I mean all of the clothes in the game are made for Monika, it's kind of a dumb question, but if they actually add it to the game I wouldn't really care.......

**Comment by lunulae** [comment]
I think you have a point about the meta side of things, that she's a drawn character in a game and she knows that. But rather than shutting down that question entirely, I think it presents a possible angle to consider the conversation from. If the player were asking so they could get Monika clothes, she might point out that she doesn't know if her body will look similar when she crosses over. That could build into an interesting discussion.
If we're framing it as a more intimate/risque inquiry, it could refer back to the topic about how the girls were designed to be appealing, someone had an intent when deciding how she should look. She might have a relationship to her body image that is more complex than a normal human's would be. I know I'd feel pretty strange about my appearance if I knew I was made to adhere to a particular beauty standard.

I suppose there is the fact that the question in itself might be objectionable to _her_, many people aren't comfortable with being that precise about the details of their appearance. I'm not sure if Monika would be shy or offended by such a thing, but it's worth considering. Keeping to Monika's character's the most important thing, after all.
But I think my major concern with the sizes question would be that if we think Monika's likely to give a straight answer, it's kind of difficult to make a guess based on the canon images we have. It's tough for me to guess someone's sizes by sight in real life, lol. I suppose there's the option that she'd dodge the question, but I'm not sure that that's preferable either.

**Comment by EllieWLMS** [comment]
You are right, but I think the question is a bit old and boring, I think we could ask more "perverted" things that are really important in a normal relationship, we could ask her about her fetishes (in a respectful way) what she thinks of the industry of porn and even what she thinks of all the adult images of her on the Internet...and again going back to the previous topic, would Monika really know her bra size? I mean ..... Monika says she doesn't remember much of her past and as I said earlier, all clothes are specially designed for Monika

**Comment by Freshairkaboom** [comment]
Yes, from her perspective, everything is still sort of 3D, otherwise she couldn't practise the piano. So she could just check.

**Comment by MonikaEndDie** [comment]
̶I̶f̶ ̶i̶t̶ ̶g̶e̶t̶ ̶i̶n̶ ̶t̶h̶e̶ ̶s̶u̶b̶j̶e̶c̶t̶,̶ ̶w̶h̶e̶n̶ ̶ ̶#̶4̶5̶0̶4̶ ̶g̶e̶t̶ ̶r̶e̶l̶e̶a̶s̶e̶d̶?̶

**Comment by WishStone-Dragon** [comment]
Isn't there the whole request from the maker of DDLC to please NOT lewd the Dokis?

I get being realistic and all, but we're already very much treading the line with some of Moni's outfits. Asking her detailed sexual questions would be walking all over the request the creator of DDLC made of us as fans.

**Reply by hatanasinclaire** [reply]
> Isn't there the whole request from the maker of DDLC to please NOT lewd the Dokis?

He almost certainly never said anything like this

**Reply by WishStone-Dragon** [reply]
Interesting. I could have sworn I read that, but after your comment, I cannot find a source. Somehow I was convinced it was even mentioned in this very mod.

**Comment by Capricus02** [comment]
Personally (and I know I don't speak for everyone) I'm not opposed to more adult topics as a part of a discussion between her and the player, but if you're wanting "perv questions" just for the sake of being pervy, you're better off getting one of the lewd submods.

Like I said though, I'm not opposed to adult topics coming up organically though like if she was curious about/asking about certain topics or things like people's obsession with comparing breast sizes or penis sizes, or her thoughts on pornography, or in case she's feeling insecure or something.

**Reply by WishStone-Dragon** [reply]
Mhm, no, I'm with you on that. Romance, snuggles, a smooch, sure, her being flirty, maybe, but I don't need MAS core Moni to be cybering me.
